src/fastify-plugins/icons.js

```diff
@@ -139,15 +147,10 @@ function assertValidSize(value) {
 
 /**
  * @param {number} value
- * @returns {asserts value is import('@mapeo/schema').Icon['variants'][number]['pixelDensity']}
+ * @returns {asserts value is import('../icon-api.js').BitmapOpts['pixelDensity']}
  */
 function assertValidPixelDensity(value) {
-  if (
-    !VALID_PIXEL_DENSITIES.includes(
-      // @ts-expect-error
-      value
-    )
-  ) {
+  if (!VALID_PIXEL_DENSITIES.includes(value)) {
```

Comment:
I would fix this error by casting `value` to `any`.

```suggestion
  if (!VALID_PIXEL_DENSITIES.includes(/** @type {any} */ (value))) {
```

Reply:
mhmh, I also needed to change the signature of the fn to  `/** @param {any} value */`